html-css/week3/index.html

```diff
@@ -0,0 +1,87 @@
+            <h1>Sign up for yourcustomtailor.com</h1>
+            <p>We can't wait to help you improve your life-style and gain the attention from people around you!</p>
+            <p>Sign up below to save up to 70% apparel, hand-bags and cosmetics. Our agents are busy picking the best
+                offers that match your preferences and style. Every week we will send you a hand-picked selection - we
+                just need to know a few details about you first...</p>
+        </div>
+        <div>
+            <h3>Step 1/4 - tell us who you are?</h3>
+            <label for="">Progress</label>
+            <progress value="20" max="100"></progress>
+            <p>Nice to meet you. Let's spend a minute to get to know each other? Please enter the following:</p>
+        </div>
+
+        <div class="info-content">
+            <div>
+                <div>
+                    <label for="name">First name*</label>
+                    <input type="text" name="name" id="name" required>
+                </div>
+                <div>
+                    <label for="country">Country</label>
+                    <select name="country" id="country" required>
+                        <option value="Denmark">Denmark</option>
+                        <option value="USA">USA</option>
+                        <option value="Ukraine">Ukraine</option>
+                    </select>
+                </div>
+                <div>
+                    <label for="address">Street address*</label>
+                    <input type="text" name="address" id="address" required>
+                </div>
+            </div>
+            <div>
+                <div>
+                    <label for="name">Last name*</label>
+                    <input type="text" name="name" id="name" required>
+                </div>
+                <div>
+                    <label for="address">Post code*</label>
+                    <input type="text" name="address" id="address" pattern="\d{4}" required>
+                </div>
+            </div>
+        </div>
+        
+        <div class="profile-description-content">
+            <div>
+                <label for="description">Profile description</label>
+                <p>Please provide a short description of yourself to be displayed on our social community pages</p>
+            </div>
+            <textarea name="description" id="description" cols="100" rows="10" pattern="\d{500}"></textarea>
+        </div>
+        <p>Use the same address for billing and shipping? Uncheck below to fill out a separate shipping address.</p>
+        <div class="submit-content">
+            <div>
+                <label for="shipping">Shipping address</label>
+                <input type="checkbox" id="shipping" name="shipping">
+            </div>
+            <button type="submit">
+                <a href="https://htmlpreview.github.io/?https://github.com/polyovik/hyf-homework/blob/week3/html-css/week3/index2.html">Next Step</a>
```

Comment:
Not sure if this was the right approach but not wrong either. It's just sometimes your page navigates to the next pages with validation and sometimes without and sometimes even fails. However, it's mainly a problem of GitHub preview. Anyway, if we put aside it and the need to preview your page in this way, then using `a` tag would be absolutely unnecessary as your form action already navigates to the next page. Keep this in mind just for the future.